packages/framework/src/Model/Product/Pricing/ProductSellingPrice.php

```diff
@@ -5,31 +5,32 @@
 namespace Shopsys\FrameworkBundle\Model\Product\Pricing;
 
 use Shopsys\FrameworkBundle\Model\Pricing\Group\PricingGroup;
-use Shopsys\FrameworkBundle\Model\Pricing\Price;
 
-class ProductSellingPrice
+final class ProductSellingPrice
 {
     /**
      * @param \Shopsys\FrameworkBundle\Model\Pricing\Group\PricingGroup $pricingGroup
-     * @param \Shopsys\FrameworkBundle\Model\Pricing\Price $sellingPrice
+     * @param \Shopsys\FrameworkBundle\Model\Product\Pricing\ProductPrice $productPrice
      */
-    public function __construct(protected readonly PricingGroup $pricingGroup, protected readonly Price $sellingPrice)
-    {
+    public function __construct(
+        private readonly PricingGroup $pricingGroup,
+        private readonly ProductPrice $productPrice,
```

Comment:
This makes me wonder if we neeed ProductPrice and ProductSellingPrice, would not be easier to have just one object for both of these use cases?

Reply:
that is a great idea, resolved in 23dde09ec1